le.go

```diff
@@ -138,9 +138,21 @@ func (logger *Logger) Flags() int {
 
 // Output does the actual writing to the TCP connection
 func (logger *Logger) Output(calldepth int, s string) error {
-	_, err := logger.Write([]byte(s))
-
-	return err
+	var err error
+	for {
+		_, err = logger.Write([]byte(s))
+		if err != nil {
+			fmt.Printf("go_le error: %v\n\tTrying to recover...", err)
```

Comment:
You shouldn't really be printing inside a library call.. same goes for the below calls to fmt.Printf()